chemistry/rosetta/pose.py

```diff
@@ -0,0 +1,79 @@
+"""
+This package contains classes responsible for loading and dumping PyRosetta
+Pose objects.
+"""
+
+from __future__ import print_function
+
+from chemistry.exceptions import RosettaError, RosettaWarning
+from chemistry.periodic_table import AtomicNum, Mass
+from chemistry.structure import Structure
+from chemistry.topologyobjects import Atom, Bond
+
+try:
+    from rosetta import Pose, AtomID
+except ImportError:
+    RosettaWarning('Cannot find the PyRosetta module.')
+
+
+class RosettaPose(object):
+
+    @staticmethod
+    def load(pose):
+        """ Load a Pose object and return a populated `Structure` class
+        """
+
+        if not isinstance(pose, Pose):
+            raise RosettaError('Object is not a PyRosetta Pose object.')
+
+        struct = Structure()
+        struct.experimental = struct.journal = struct.authors = ''
+        struct.keywords = struct.doi = struct.pmid = ''
+        struct.journal_authors = struct.volume_page = ''
+        struct.title = ''
+        struct.year = None
+        struct.related_entries = []
```

Comment:
ParmEd makes no guarantees that this metadata exists -- I had need for some of the PDB metadata in the past, so I added it to my PDB parser and just stuck on those attributes.  Only the PDB and CIF parsers add this metadata, so I don't think it's necessary here.